django/template/defaulttags.py

```diff
@@ -578,10 +615,14 @@ def __repr__(self):
         return "<WithNode>"
 
     def render(self, context):
+        return ''.join(self.stream(context))
```

Comment:
Yes, it allows tags which don't need stream logic to not have to implement it. Changing it would also break all existing custom tags.